internal/generator/crypto/hash/mimc/template/mimc.go.tmpl

```diff
@@ -89,7 +89,6 @@ func (d *digest) Write(p []byte) (n int, err error) {
 // The XOR operation is replaced by field addition, data is in Montgomery form
 func (d *digest) checksum() fr.Element {
 
-	var buffer [BlockSize]byte
 	var x fr.Element
```

Comment:
@ThomasPiellard this variable x is always zero; yet it appears in the loop below as `d.h.Add(&r, &d.h).Add(&d.h, &x)` is this a mistake?